testcharms/charm-repo/bionic/dummy/hooks/remove

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Let's add a stop hook as well, to ensure that stop gets called before remove.  I did so for my testing.

Reply:
Added